abusiveexperiencereport/v1/abusiveexperiencereport-gen.go

```diff
@@ -259,6 +259,9 @@ type SitesGetCall struct {
 }
 
 // Get: Gets a site's Abusive Experience Report summary.
+//
+// - name: The name of the site whose summary to get, e.g.
+// `sites/http%3A%2F%2Fwww.google.com%2F`. Format: `sites/{site}`
```

Comment:
I wonder if we should add a period at the end if one isn't there?

Reply:
That is a good idea!